Exercise_1.java

```diff
@@ -1,46 +1,85 @@
+import java.util.EmptyStackException;
+
+import org.w3c.dom.ls.LSException;
+
+// Time Complexity : O(1)
+// Space Complexity : O(n)
+// Did this code successfully run on Leetcode : N/A, worked on test cases
+// Any problem you faced while coding this : throw EmptyStackException instead of returning 0 - is this correct?
+

```

Comment:
IMO, for this problem it is acceptable but ideally, it would have been specified what should be returned ( maybe `null` ).